.github/workflows/build.yml

```diff
@@ -70,22 +70,6 @@ jobs:
           kill $(cat server.pid)
           rm server.pid
 
-      - name: Remove latest installed platform version
```

Comment:
I understand that these steps are not really necessary as they are taking care by the arduino-cli installation or?

Reply:
Thats right! We don't need to do any copy and linking here. This approach is working very well and stable. I guess we can take this workflow also for XMC.